libcodechecker/server/api/report_server.py

```diff
@@ -1719,6 +1719,15 @@ def getMissingContentHashes(self, file_hashes):
     def massStoreRun(self, name, tag, version, b64zip, force):
         self.__require_store()
 
+        try:
+            session = self.__Session();
+            run = session.query(Run).filter(Run.name == name).one_or_none()
+
+            if run and self.__storage_session.has_ongoing_run(run.name):
+                raise Exception('Storage of ' + name + ' is already going')
```

Comment:
Do we handle this nicely at the client side?
A message to the user would be nice that there is already an ongoing storage process.

Reply:
Yes, the message is printed here: https://github.com/Ericsson/codechecker/blob/master/libcodechecker/libhandlers/store.py#L331

Reply:
Write an exclamation mark at the end of the sentence.